### ISSUE TITLE
TextField Detail Label not Layed-Out correctly with Left-Image

### ISSUE THREAD

**Issue by cody1024d** [CosmicMind/Material#1053]
So, I ended up tracking down this issue after my app is not displaying the entirety of the detail label when there is a left image.  Specifically, I believe there is an issue in the following code:

```swift
    /// Layout the detailLabel.
    func layoutDetailLabel() {
        let c = dividerContentEdgeInsets
        detailLabel.frame.size.height = detailLabel.sizeThatFits(CGSize(width: bounds.width, height: .greatestFiniteMagnitude)).height
        detailLabel.frame.origin.x = c.left
        detailLabel.frame.origin.y = bounds.height + detailVerticalOffset
        detailLabel.frame.size.width = bounds.width - c.left - c.right
    }
```

The issue is the following line, specifically:

```swift
let c = dividerContentEdgeInsets
        detailLabel.frame.size.height = detailLabel.sizeThatFits(CGSize(width: bounds.width, height: .greatestFiniteMagnitude)).height
```

We're taking the bounds of the entirety of the textfield (which includes the left image's width) and using that to compute the height of the detail label.  Unfortunately, this is incorrect, because the detail label does not go across the full width of the view, it follows the width of the actual textfield only.

So in my case, I'm getting an incorrect height, because my detail text COULD fit in one line (height of 16) if the detail label stretched the whole width, HOWEVER, since this isn't the case, I really need a height of 32.  I believe the correct fix is to use detailLabel.frame.width there, as that's the actual width of the label that we're sizing.

I've made the changes locally, so I've had to essentially detatch myself from updates for this class until this is fixed :(

**Comment by danieldahan** [comment]
Hey! Thank you for finding this issue. I'll look into this today and make a release.

**Comment by danieldahan** [comment]
I haven't finished my investigation on this. Will update you shortly. Thank you!

**Comment by OrkhanAlikhanov** [comment]
Hey @danieldahan, this is the issue we discussed recently. It should've been solved in #1082

**Comment by danieldahan** [comment]
Excellent, so let's make a fix for this, @OrkhanAlikhanov would you be able to PR a fix in?
